### ISSUE TITLE
Allegro Kurier UPS - błąd wielopaczkowości?

### ISSUE THREAD

**Issue by ZAK28ZAK** [allegro/allegro-api#9773]
Dzień dobry,

Dla podanego trace ID=3e40cb62aee47173

nie możemy wygenerować przesyłki wielopaczkowej, czy ta metoda nie obsługuje wielopaczkowości czy coś źle przekazujemy w zapytaniu API?

**Comment by MartaNowaczyk** [comment]
Dzień dobry, UPS nie obsługuje wielopaczkowości.

**Comment by MartaNowaczyk** [comment]
Nie, na ten moment nie mamy nawet takiej możliwości, bo to jeszcze jest zależne od przewoźnika.
